projects/igniteui-angular/src/lib/grids/tree-grid/tree-grid-integration.spec.ts

```diff
@@ -277,4 +287,231 @@ describe('IgxTreeGrid - Integration', () => {
             expect(parseInt(column.width, 10)).toBe(152);
         });
     });
+
+    describe('Row editing', () => {
+        beforeEach(() => {
```

Comment:
Add configureTestSuite().

Reply:
Not needed unless the test suite calls `TestBed.configureTestingModule(/*...*/).compileComponents();`